### DISCUSSION TITLE
华为nova4打不开

### DISCUSSION THREAD

**Discussion by apkpai** [Kuingsmile/PicHoro#29] [General]
软件安装打开后直接灰屏
截图：http://markdown.****.cn/markdown_imgs/1668172099784.jpg

**Comment by Kuingsmile** [comment]
> 软件安装打开后直接灰屏 截图：http://markdown.apkpai.cn/markdown_imgs/1668172099784.jpg

你好，我手头没有nova4的机子，用我老婆的nova9试了一下是可以正常安装和打开使用的，请您尝试一下在权限管理中把存储，相册访问等功能设置成始终允许或使用应用时允许试试，本软件需要的权限有：
- 网络连接权限
- 存储：允许管理所有文件
- 相机权限
- 访问和写入剪贴板权限
- 使用摄像头权限
- 安装未知应用权限

另外，我测试用的nova9的系统为鸿蒙2.0.1，如下图所示：
![e4908c1ae1cdc7f8b1f844b45c80add](https://user-images.githubusercontent.com/96409857/201356054-0bd76fb1-1472-4892-a507-ec4bfa713cfd.jpg)

**Reply by apkpai** [reply]
好的，我刚才又试了一下，给予所有权限之后可以正常使用了。
系统：emui，Android10
